### PR TITLE
Closes #63 - Add regular expression for Password field in UserDTO

### LINKED ISSUE #63: Add regular expression for Password field in UserDTO
Valid password: at least 8 symbols, among which at least one lowercase letter, one Uppercase letter, one number and one special character, e.g. @, !, etc

### PR DESCRIPTION
**Pull request by nataliapachva**
Added regular expression for Password field in UserDTO
Changed MinLength to 8 symbols for Password field

### CODE REVIEW

File: Common/ModelsDTO/UserDTO.cs [hunk 1]
```diff
@@ -10,10 +10,10 @@ public class UserDTO
         public string LastName { get; set; }
         [Required]
         [EmailAddress(ErrorMessage = "Invalid Email Address")]
-        
         public string Email { get; set; }
         [Required]
-        [MinLength(6), MaxLength(16)]
+        [MinLength(8)]
+        [RegularExpression(@"^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=.,\-_!])([a-zA-Z0-9 @#$%^&+=*.,\-_!]){8,}$", ErrorMessage = "Password invalid.")]
```
Comment (roman-fay) [inline Common/ModelsDTO/UserDTO.cs]: 'Password is invalid' + put this string to Constants class

Comment (nataliapachva) [reply Common/ModelsDTO/UserDTO.cs]: Done

File: Common/ModelsDTO/UserDTO.cs [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
         [Required]
-        [EmailAddress(ErrorMessage = "Invalid Email Address")]
-        
+        [EmailAddress]
         public string Email { get; set; }
+
         [Required]
-        [MinLength(6), MaxLength(16)]
+        [MinLength(8)]
+        [RegularExpression(@"^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=.,\-_!])([a-zA-Z0-9 @#$%^&+=*.,\-_!]){8,}$", ErrorMessage = Constants.PasswordInvalidMessage)]
```
Comment (roman-fay) [inline Common/ModelsDTO/UserDTO.cs]: actually, put regex to Consts as well..

Comment (nataliapachva) [reply Common/ModelsDTO/UserDTO.cs]: Done

**Pull request merged**